developer/src/kmcmplib/tests/get_test_source_exclude_some_kmn.bat

```diff
@@ -17,5 +17,5 @@ rem		- vietnamese_vni
 
 rem added CERR_XXXX-Tests to test if right ErrorMessage is sent
 
-dir  /s/b ..\..\..\..\common\test\keyboards\invalid\*.kmn
-dir  /s/b ..\..\..\..\..\keyboards\release\*.kmn | more |   findstr /v /i  "viet*" |  findstr /v /i  "eKwTamil99UniUpdt"
+dir  /s/b %KEYMAN_ROOT%\common\test\keyboards\invalid\*.kmn
+dir  /s/b %KEYMAN_ROOT%\..\keyboards\release\*.kmn | more |   findstr /v /i  "viet*" |  findstr /v /i  "eKwTamil99UniUpdt"
```

Comment:
This doesn't look right. Should this be `%KEYMAN_ROOT%\common\test\keyboards\release\*.kmn` ?

Reply:
see above comment